js/components/main.js

```diff
@@ -224,11 +224,11 @@ class Main extends ImmutableComponent {
       <div className='top'>
         <div className='navigator-wrapper'>
           <div className='backforward'>
-            <span
+            <span title='Go back'
```

Comment:
`app/locales/en-US/app.l20n`
Example:
`<backButton
  title: "Go back">`

Then set the data-l10n-id